.circleci/config.yml

```diff
@@ -41,6 +41,13 @@ jobs:
             sudo make install
             sudo ldconfig # Refresh cache
             popd
+      - run:
+          name: Check virtual method offset
+          command: |
+            source $HOME/dlang/ldc-1.20.1/activate
+            ci/check_vtable_test.d
```

Comment:
By the way, we could have added this check into `run.sh` instead? Is there a reason why it was moved to CircleCI?

Reply:
https://github.com/bpfkorea/agora/pull/711#pullrequestreview-386915944
I just didn't want to have a big impact on Agora's development environment.